storage/pebble/registers/payload/lookup.go

```diff
@@ -0,0 +1,87 @@
+package payload
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage/pebble/registers/config"
+)
+
+// lookupKey is the encoded format of the storage key for looking up register value
+type lookupKey struct {
+	encoded []byte
+}
+
+// newLookupKey takes a height and registerID, returns the key for storing the register value in storage
+func newLookupKey(height uint64, reg flow.RegisterID) *lookupKey {
```

Comment:
I'm pretty sure we will need to add here a prefix for `registers`. So we can have multiple entities in the future in the same db.

Reply:
+1